orchestrator/graphql/extensions/deprecation_checker_extension.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022-2023 SURF.
```

Comment:
Preferably we also move and publish this to a separate pypi package before it starts to appear in 3 or more places

Reply:
will be done in a different PR, don't want to keep this hanging.

Reply:
Tag me on that issue please :)